collator/src/lib.rs

```diff
@@ -447,6 +453,52 @@ fn run_collator_node<S, P, Extrinsic>(
 	Ok(service)
 }
 
+/// Async function that will run the collator node with the given `RelayChainContext` and `ParachainContext`
+/// build by the given `BuildParachainContext` and arguments to the underlying polkadot node.
```

Comment:
```suggestion
/// built by the given `BuildParachainContext` and arguments to the underlying polkadot node.
```

Reply:
wow I copy-pasted this typo 3 times from the original comment without seeing it, thanks!